examples/ex14.cpp

```diff
@@ -18,6 +18,8 @@
 //               ex14 -m ../data/amr-quad.mesh -r 3
 //               ex14 -m ../data/amr-hex.mesh
 //               ex14 -m ../data/fichera-amr.mesh
+//               ex14 -m ../data/inline-quad.mesh -r 2 -rk
```

Comment:
Are these correct?

./ex14 -m ../data/inline-quad.mesh -r 2 -rk
Options used:
   --mesh ../data/inline-quad.mesh
   --refine 2
   --order 1
   --rk
   --sigma 1
   --kappa 4
   --eta 0
   --visualization
Number of unknowns: 4096

Assertion failed: (FE->GetOrder() == fec->GetOrder()) is false:
 --> internal error: 8 != 0
 ... in function: virtual const mfem::FiniteElement* mfem::FiniteElementSpace::GetFE(int) const
 ... in file: /mnt/home/develop/wrk/rbf/mfem/fem/fespace.cpp:2824

Aborted (core dumped)